src/sagemaker_training/process.py

```diff
@@ -40,6 +40,29 @@
 _DEFAULT_BUF_SIZE = 1024 * 64
 
 
+def get_debugger_exception_classes():
+    """Set exception classes"""
+    exception_classes = []
+    if os.environ.get("USE_SMDEBUG") == "0":
+        logger.info(
+            "sagemaker debugger is not enabled, using exception types without debugger ones."
+        )
+    else:
+        try:
+            from smdebug import exceptions
+
```

Comment:
remove extra line.

Reply:
this extra line is from black check and so can't remove it.